app-laptop/laptop-mode-tools/laptop-mode-tools-1.74.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils multilib systemd udev
+
+MY_P="${PN}_${PV}"
+
+DESCRIPTION="Linux kernel laptop_mode user-space utilities"
+HOMEPAGE="https://github.com/rickysarraf/laptop-mode-tools/wiki"
+SRC_URI="https://github.com/rickysarraf/${PN}/releases/download/${PV}/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="+acpi apm"
+
+RDEPEND="sys-apps/iproute2
+	sys-apps/ethtool
+	sys-apps/which
+	|| (
+		sys-apps/sdparm
+		sys-apps/hdparm
+	)
+	acpi? ( sys-power/acpid )
+	apm? ( sys-apps/apmd )"
+
+S="${WORKDIR}/${MY_P}"
+
+src_compile() { :; }
+
+src_install() {
+	DESTDIR="${D}" \
+		INIT_D="none" \
+		MAN_D="/usr/share/man" \
+		LIB_D="/usr/$(get_libdir)" \
+		UDEV_D="$(get_udevdir)" \
+		SYSTEMD_UNIT_D="$(systemd_get_systemunitdir)" \
+		TMPFILES_D="/usr/lib/tmpfiles.d" \
+		ACPI="$(usex acpi force disabled)" \
+		PMU="disabled" \
+		APM="$(usex apm )" \
+		SYSTEMD=yes \
+		sh ./install.sh || die
+
+	dodoc Documentation/*.txt README.md
+	newinitd "${FILESDIR}"/laptop_mode.init-1.4 laptop_mode
+
+	rm "${ED}/usr/lib/pm-utils/sleep.d/01laptop-mode" || die "could not remove PM utils config."
+}
+
+pkg_postinst() {
+	local daemon_name
+	optfeature "bluetooth support" net-wireless/bluez
+	if use acpi || use apm; then
+		if use acpi; then
+			daemon_name="acpid"
+		elif use apm; then
+			deamon_name="apmd"
+		fi
+		if [ "$(rc-config list default | grep laptop_mode)" = "" ] || [ "$(rc-config list default | grep ${daemon_name} )" = "" ]; then
+			ewarn "To enable automatic power state event handling,"
+			ewarn "e.g. enabling laptop_mode after unplugging the battery,"
+			ewarn "both laptop_mode and the ${daemon_name} daemon must be"
+			ewarn "added to default runlevel:"
+			if [ "$(rc-config list default | grep laptop_mode)" = "" ]; then
```

Comment:
Then we can simplify the whole thing. No need to be checking rc-config output etc.

Reply:
Done!